client/src/main/java/io/pravega/client/admin/impl/StreamManagerImpl.java

```diff
@@ -104,6 +131,49 @@ public boolean deleteScope(String scopeName) {
                 RuntimeException::new);
     }
 
+    @Override
+    public StreamInfo getStreamInfo(String scopeName, String streamName) {
+        NameUtils.validateScopeName(scopeName);
```

Comment:
The implementation of this should be shared with the existing impl that is currently in the BatchAPI. I don't want to have two copies of this code.

Reply:
done, reused the implementation that is present in BatchAPI.